lib/logproto/tests/test-text-server.c

```diff
@@ -360,7 +360,7 @@ accumulator_assert_that_lines_are_starting_with_sequence_number(LogProtoTextServ
 {
   assert_true((msg[0] - '0') == accumulate_seq,
               "accumulate_line: Message doesn't start with sequence number, msg=%.*s, seq=%d",
-              msg_len, msg, accumulate_seq);
+              (int)msg_len, msg, accumulate_seq);
```

Comment:
The `gsize` is probably a `long`, you should not cast to `int` because of printing. I would use the `G_GSSIZE_FORMAT` format string, which has the proper format string to this type.

Reply:
In this case, `msg_len` is not printed, it just sets the precision of the `msg` string.
The precision can only be given as an `int`, this is why I only casted it. Alternatively we could get rid of the precision, or convert `msg_size` to int with a safer routine. Which one would you prefer? Thanks!

Reply:
Sorry, my bad. In that case the `(int)` conversion is fine.